src/resource.ts

```diff
@@ -586,7 +586,6 @@ export abstract class BaseResource<
                         this.prepareCredentialsFilter(this.callerSession)
                     );
                 }
-                this.log('EVENT DATA\n', eventData);
```

Comment:
is that the only place?

Reply:
Yes, this is the only place logging event data in the repo.

Reply:
Validated this is the only place logging event data by invoking CUD handlers of a resource.